runtime/src/weights/pallet_author_mapping.rs

```diff
@@ -32,37 +32,25 @@ pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_author_mapping::weights::WeightInfo for WeightInfo<T> {
     // Storage: AuthorMapping MappingWithDeposit (r:1 w:1)
     // Storage: System Account (r:1 w:1)
+	#[rustfmt::skip]
     fn add_association() -> Weight {
-        (72_930_000 as Weight)
-            .saturating_add(T::DbWeight::get().reads(2 as Weight))
-            .saturating_add(T::DbWeight::get().writes(2 as Weight))
-    }
+		(34_696_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(2 as Weight))
+			.saturating_add(T::DbWeight::get().writes(2 as Weight))
+	}
```

Comment:
[nitpick] A couple of tabs snuck in here. I'm okay with that in auto-generated files, but I thought I'd mention it.

Reply:
I will remove them in the next and last PR before v0.3.2 #627